gallery/javascript-comments-to-code/mathematical-operation-javascript.js

```diff
@@ -0,0 +1,29 @@
+//addition of two numbers
+function addition(a, b) {
+    return a + b;
+}
+
+//subtraction of two numbers
+function subtraction(a, b) {
+    return a - b;
+}
+
+//multiplication of two numbers
+function multiplication(a, b) {
+    return a * b;
+}
+
+//division of two numbers
+function division(a, b) {
+    return a / b;
+}
+
+//determine if it's a prime number
+function isPrime(num) {
+    for (var i = 2; i < num; i++) {
+        if (num % i === 0) {
+            return false;
+        }
+    }
+    return num > 1;
+}
```

Comment:
```suggestion
// addition of two numbers
function addition(a, b) {
    return a + b;
}

// subtraction of two numbers
function subtraction(a, b) {
    return a - b;
}

// multiplication of two numbers
function multiplication(a, b) {
    return a * b;
}

// division of two numbers
function division(a, b) {
    return a / b;
}

// determine if it's a prime number
function isPrime(num) {
    for (var i = 2; i < num; i++) {
        if (num % i === 0) {
            return false;
        }
    }
    return num > 1;
}
```
I'm pretty sure it's convention to add spaces between the // and the comment when writing comments, but other than that, this LGTM!

Reply:
Yah! Thanks. I'll add that to commit.

Reply:
:)